tests.nix

```diff
@@ -0,0 +1,14 @@
+{ packages, system, pkgs }:
+if pkgs.lib.strings.hasSuffix "darwin" system
```

Comment:
You can't run nixos tests on darwin because it uses `qemu` for virtualization. This issue https://github.com/NixOS/nixpkgs/issues/108984 tracks the progress on this problem.